Gopkg.lock

```diff
@@ -1,6 +1,12 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
 
 
+[[projects]]
```

Comment:
can you check in Gopkg.toml as well? this lock file should be derived from `Gopkg.toml`.